doc/utilities/asc_to_qsch.rst

```diff
@@ -1,16 +1,16 @@
-asc_to_qsch.exe
+asc_to_qsch
```

Comment:
This is really a .exe file that is installed by the Pip installer when the PyPi version is used.